scripts/run-vscode-integration-tests-with-top-level-extensions.js

```diff
@@ -10,7 +10,7 @@ const cwd = process.cwd();
 // Executes the test, using the top-level packages as the CODE_EXTENSIONS_PATH
 
 shell.exec(
-  `cross-env CODE_EXTENSIONS_PATH='${path.join(
+  `cross-env CODE_VERSION='1.30.0' CODE_EXTENSIONS_PATH='${path.join(
```

Comment:
Spinning up vscode 1.31.1 with all the code under `packages` is causing it to crash in Windows environments, which is what we have appveyor running on. I'm setting it to use `1.30.0` while I find a fix for this. FYI, linux and macos builds of vscode do not crash.

Reply:
Does this mean we're going to have to update this as the code version changes?

Reply:
No, this is a stop gap while we figure out how to run these tests in the latest vscode version. Also, not all test suites run using this script, most are still running on the latest stable vscode version (currently 1.31.1).